code/modules/clothing/head/helmet.dm

```diff
@@ -42,15 +42,21 @@
 	name = "head of security hat"
 	desc = "The hat of the Head of Security. For showing the officers who's in charge."
 	icon_state = "hoscap"
-	armor = list(melee = 65, bullet = 30, laser = 50, energy = 10, bomb = 25, bio = 0, rad = 0)

```

Comment:
Put this back.

Reply:
No. It is a hat, not a helmet.

Reply:
Yes. It is a unique piece of gear for the HoS.